### DISCUSSION TITLE
Relicense under Apache-2.0

### DISCUSSION THREAD

**Discussion by MikeRalphson** [OAI/sig-moonwalk#48] [General]
As per the OAS itself, OAI spec work should be Apache-2.0 licensed AFAIK. Moonwalk currently has a MIT license - though this may have just been an oversight. IANAL when it comes to MIT->Apache-2.0 relicensing but I feel it would be better to change the license as soon as possible to prevent any concerns if and when content is moved to the main repo.

**Comment by darrelmiller** [comment]
That was an oversight on my part. I'll fix it.

**Reply by darrelmiller** [reply]
Done.

**Comment by handrews** [comment]
This has been done, closing!
